examples/plot_draw_legend_docs.py

```diff
@@ -0,0 +1,107 @@
+"""
+Custom Legends with EarthPy
+===========================
+
+
+"""
+
+###############################################################################
+# Plotting with EarthPy
+# ---------------------
+#
+# .. note:: The examples below will show you how to make use of the draw_legend function for creating plots
+
+
+###############################################################################
+# Plot Continuous Data
+# ------------------------------
+#
+# Let's explore a simple plot using earthpy. To begin, import the needed packages
+# and create an array to be plotted. Below we plot the data as continuous with a colorbar
+# using the ``plot_bands()`` function
+
+
+import matplotlib.pyplot as plt
+import earthpy.plot as ep
+import numpy as np
+
+# Create a numpy array. Let's pretend this is what you want to plot.
+arr = np.random.randint(4, size=(5, 5))
+
+# When plot_bands is updated a cbar will be here as well
+ep.plot_bands(arr)
+plt.show()
+
+###############################################################################
+# Create Custom Discrete Legends with Earthpy
+# If you want to create a custom categorical legend, you can use the ``ep.draw_legend()`` function.
+
+f, ax = plt.subplots()
+im = ax.imshow(arr)
+ep.draw_legend(im)
+plt.show()
+
+###############################################################################
+# Discrete Legends
+# ------------------------------
+# By default the draw_legend function creates a legend with default categories.
+# You can customize the legend by adding titles
+
+f, ax = plt.subplots()
+im = ax.imshow(arr)
+ep.draw_legend(im, titles=["Small", "Bigger", "Even Bigger", "Ginormous"])
+plt.show()
```

Comment:
The legend here is also cut off

![image](https://user-images.githubusercontent.com/2664564/55168373-5229cb80-5138-11e9-8b72-0f02f106caaa.png)

Reply:
All of the legends are cut off. again i haven't been able to figure out how to make sure they plot properly. i haven't spent a lot of time on it either but figsize doesn't work.